src/features/partners/ManagementToolbar.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonLink from '../../common/ButtonLink';
+import ButtonLinkGroup from '../../common/ButtonLinkGroup';
+
+export default function ManagementToolbar({ partner, selectedTab }) {
+  return (
+    <div className="row">
+      <div className="col col-9">
+        <ButtonLinkGroup>
+          <ButtonLink className={selectedTab === 'cohorts' ? 'btn btn-dark' : 'btn btn-outline-dark'} link={`/${partner}/admin`}>
```

Comment:
Idea for later: A future iteration could use a `selected` boolean prop instead of the parent passing `className`.

Reply:
Is this something we want to implement now as opposed to later?

Reply:
Decided to take a crack at this approach. I think I managed to make a decent attempt.